### PR TITLE
Eslint: don't allow var

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: functional-samples/sample.bookmarks/popup.js [hunk 1]
```diff
@@ -51,7 +52,7 @@ function dumpNode(bookmarkNode, query) {
       chrome.tabs.create({ url: bookmarkNode.url });
     });
 
-    var span = $('<span>');
+    span = $('<span>');
```
Comment (patrickkettner) [inline functional-samples/sample.bookmarks/popup.js:55]: Why is this an empty string redefined? Is there a reason it can't just be a const?

Comment (sebastianbenz) [reply functional-samples/sample.bookmarks/popup.js:55]: scope

**Review by patrickkettner — APPROVED** [review]

File: functional-samples/sample.bookmarks/popup.js (hunk 1 above)
Comment (sebastianbenz) [reply functional-samples/sample.bookmarks/popup.js:55]: yeah, const was also my first try. eslint is better at these things than us humans

Comment (patrickkettner) [reply functional-samples/sample.bookmarks/popup.js:55]: 🤖

**Pull request merged**